### ISSUE TITLE
forEachParallel lost input function name information 

### ISSUE THREAD

**Issue by leitsubomi** [davepacheco/node-vasync#26]
When forEachParallel expand task function from arguments array , it losts function name, even the task function has name. Please see vasync.js line 135.  It will be hard to debug.  Please kindly fix it in your convenience.

**Comment by davepacheco** [comment]
Yeah, this looks to be an artifact of the way we use parallel() to implement forEachParallel().
